### ISSUE TITLE
UX: Accessibility

### ISSUE THREAD

**Issue by techsmyth** [alkem-io/client-web#2316]
## Description
As usage of Alkemio grows,the platform needs to be accessible by users that may be visually or otherwise impaired 

## Initiative / goal
Accessibility

### Hypothesis
By making the platform more accessible, that more users will engage and Alkemio will wider used by public sector entities.

## Acceptance criteria and must have scope
- [ ] Navigation by tabs properly supported
- [ ] Anchors are properly defined
...

## Stakeholders
UWV 
EnVision

## Timeline
tbd

## Additional Information

**Review UWV Innovation Platform for accessibility**
Tested in Edge with keyboard only and with VoiceOver reading software.

**Keyboard navigation explanation**
Web visitors who cannot navigate with the mouse, such as people with a visual or motor impairment, usually use the keyboard for this. They do this with the tab key. By using tab and shift + tab they go to the next or previous link or button on the page. Which element is active (the tab focus), becomes visible because there is a frame around it, the so-called 'focus rectangle'.

**Tab focus**
One of the most common problems with keyboard control is the poor visibility (or complete lack) of focus indication on the active element. By default, a browser provides a delineation around the active element when operating keyboards. These indicators range from a blue and white circle to a dotted line in different browsers. There are also opportunities for users to customize these styles as needed. A common mistake is that such prompts are disabled via CSS with an outline: 0; or outline: none; value.

**Tab order**
It is also important in what order interactive items on the page get focus. The user usually expects this to happen from left to right and from top to bottom.

**Skiplink**
A skiplink is typically the first tab position of a page and is intended to skip blocks of recurring content (navigation) to go directly to the content of the page.

**Research page**
Tested page: https://hub.alkem.io/uwv/dashboard

**Skiplink**
There is no skiplink on the page.

**Tab focus**
5th tab "SIGN IN" and 6th tab "LOG IN", 7th tab "HOME", 8th tab "CHALLENGES", 9th tab "CONTRIBUTORS" 10th tab (unknown) have no visible focus indication.
The tab navigation (tab group 'Hub tabs') "OVERVIEW, CONTEXT, COMMUNITY, CONTRIBUTIONS, CHALLENGES and CANVASSES" are not included in the tab order (tabindex="-1"). Tabging takes you from 'UWV INNOVATIEPLATFORM' directly to the button 'LOG IN TO REGISTER'.  Incidentally, the tab group is  accessible for reading software (VoiceOver) with the VO + arrow key combination.
WCAG 2.4.7: Focus visible

**Heading order**
Normally, a web page has one H1 heading, the most important title of a page. This is missing from the page. An h1 is followed hierarchically by one or more h2 headings, which in a special case can be followed by an h3 heading , etc..
There are no heading h1, h2 or h3 on this page. H4 and h5 headings are used.
Heading map of available headings:
•	h4: 'UWV Innovation Platform'
•	h4: 'Hub statistics'
•	h4: 'Facilitating organisation'
o	h5: 'UWV'
•	h4: 'Aspects (12)'
WCAG 1.3.1: Info and relationships

**Use links or buttons**
To navigate to other (parts of) pages we use links `<a>`, to start an action a button `<button>`.
On the page there are three functions that can be seen as a button, 'WRITE IN', 'LOG IN' and LOG IN TO REGISTER'. However, these three buttons were not created correctly with the <button tag> but were created as a link to the html tag `<a>`.
WCAG 1.3.1: Info and relationships

**Link texts**
A link text must be self-explanatory . Users of reading software hear the text of the link outside the context of the story. As a result, link texts such as 'Read more', 'See everything' but also texts such as 'Click here' or 'Read more' are not correct. 
WCAG 2.4.4: Link purpose (in context)

**Use CAPS**
On the page, texts of buttons and links are written out in capital letters. This can be an obstacle for people with reading difficulties (e.g. dyslexi, low literacy). 

**ALT texts**
An ALT text (also known as ALT tags) is an invisible description of an image. This piece of text provides information to website users with a visual impairment about what can be seen on an image. The reading equipment used by the visually impaired reads what is described in the ALT text. This way the user also knows what is on an image. When the image does not add anything to your page, for example because it is an atmospheric image, an ALT text is not always necessary and can even cause a lot of distraction. For accessibility, you do not have to place ALT text.

On the page there is an image with the alt text 'Banner'. This is not an accurate description of what can be seen. If the image is decorative, the alt text may also be left blank (alt="").

**Explanation reading software navigation**
Reading software, also called a screen reader, is a program that reads text on the screen using a synthetic voice. A screen reader not only reads text aloud, but also names elements of the interface or web page. Think, for example, of a window, a button, header above a text or a list. Furthermore, events such as notifications are automatically read aloud.

A screen reader can also read keyed-in letters and words aloud during input. As a result, the user immediately hears the result.

**Operation of reading software**
A screen reader approaches a web page linearly. That means it starts at the top of the page and reads from left to right. As if it were a text document. In addition, the html code determines  the order. That's why the correct use of html code is  so important.

As soon as you start a screen reader, it first reads the title of the web page and then it starts with the other text. All the elements that you can pronounce are mentioned, including the headings and buttons, tables and labels, input fields and descriptions of images.

**Frequently used reading software**
JAWS (used within UWV and provided by UWV Facilities)
NVDA (free reading software)
VoiceOver (comes standard on Apple laptops, PCs, tablets, and smartphones)

**Tools for navigating with reading software**
Users of reading software can scan a web page in various ways.
•	Navigating by heading structure
•	Navigate with keyboard shortcuts
•	Navigating with links

Research page
Tested page: https://hub.alkem.io/uwv/dashboard

**Tabbed navigation**
With the navigation by means of the tab key you jump from interactive element to interactive element. 
•	Logo Alkemio: missing an alto text "Home | Logo Alkemio". Links to https://hub.alkem.io. When you activate this link, a new screen opens but the user is not informed that a screen change has taken place.
•	Search: tab key takes you to the input field. When the user navigates through the page with special keys (VO+right arrow), you will first enter the 'Search group', then the text field 'Search' and then 'end of Search group'. When the user enters a search term and clicks ENTER, you will be taken to a search results page, but the user will not be informed about screen switching.
•	Language selection list. For sighted user, there is a globe icon to explain that this is a language choice. For visually impaired users, there is no further explanation (aria label) that this is a language choice. The choice 'and' is read out as "and not selected", so it is not clear that this is the language choice English. The other choices are read out as separate letters, n l, b g, u a.
•	The help link opens a new page but the user is not informed about screen switching.
•	The link "SUBSCRIBE" is read as "Sign up". This is not correct, the text that describes the interface are at least the same as the text shown. 
WCAG 2.5.3. Label in name
•	The "LOG IN" link is read as "Sign in". This is not correct, the text that describes the interface must be at least the same as the text shown. 
WCAG 2.5.3. Label in name
•	The10th tab goes to the logo at the top of the alkemio page, which is also the 1st tab. This does not make sense.
•	The Hub tabs tab group is not accessible via the TAB key, only via the reading software (VoiceOver) VO + arrow key combination. See also earlier comment.
•	The card under the heading 'Facilitating organisation' is read out as: "link, associated-organization-card". This is very unclear.
•	The last tab position is a hidden button that only becomes visible when you scroll all the way down and is to scroll back up. It is an icon that points upwards. It has no explanatory name but is called "Button". This is not correct.

**Navigation through all elements (VO+right arrow )**
Only deviations that have not been mentioned before.
•	Is the only way to access the 'Hub tabs' tab group .
•	The paragraph under the heading 'Facilitating organisation cannot be read
